enterprise/server/backends/s3_cache/s3_cache.go

```diff
@@ -39,15 +41,15 @@ var (
 	bucket                   = flag.String("cache.s3.bucket", "", "The AWS S3 bucket to store files in.")
 	pathPrefix               = flag.String("cache.s3.path_prefix", "", "Prefix inside the AWS S3 bucket to store files")
 	credentialsProfile       = flag.String("cache.s3.credentials_profile", "", "A custom credentials profile to use.")
-	ttlDays                  = flag.Int64("cache.s3.ttl_days", 0, "The period after which cache files should be TTLd. Disabled if 0.")
+	ttlDays                  = flag.Int("cache.s3.ttl_days", 0, "The period after which cache files should be TTLd. Disabled if 0.")
 	webIdentityTokenFilePath = flag.String("cache.s3.web_identity_token_file", "", "The file path to the web identity token file.")
 	roleARN                  = flag.String("cache.s3.role_arn", "", "The role ARN to use for web identity auth.")
 	roleSessionName          = flag.String("cache.s3.role_session_name", "", "The role session name to use for web identity auth.")
 	endpoint                 = flag.String("cache.s3.endpoint", "", "The AWS endpoint to use, useful for configuring the use of MinIO.")
 	staticCredentialsID      = flag.String("cache.s3.static_credentials_id", "", "Static credentials ID to use, useful for configuring the use of MinIO.")
 	staticCredentialsSecret  = flagutil.New("cache.s3.static_credentials_secret", "", "Static credentials secret to use, useful for configuring the use of MinIO.", flagutil.SecretTag)
 	staticCredentialsToken   = flag.String("cache.s3.static_credentials_token", "", "Static credentials token to use, useful for configuring the use of MinIO.")
-	disableSSL               = flag.Bool("cache.s3.disable_ssl", false, "Disables the use of SSL, useful for configuring the use of MinIO.")
+	disableSSL               = flagutil.New("cache.s3.disable_ssl", false, "Disables the use of SSL, useful for configuring the use of MinIO.", flagutil.DeprecatedTag("Specify a non-HTTPS endpoint instead."))
```

Comment:
I don't have a lot of context for this flag, but typically "deprecated" means something still works but is slated for removal in a future version. In this case it looks like `disableSSL` will do nothing at all. If the flag is no longer functional, it would be better to error out on initialization instead of silently ignoring the flag. 
@siggisim Do you have any context on how many people might be relying on this flag and if there's anyone we should warn ahead of time?

Reply:
Makes sense. @tempoz Can you remove the mention of `disable_ssl` from the docs Siggi linked?

Reply:
Done!